anw/TD-ANWv2024.1.md

```diff
@@ -0,0 +1,746 @@
+  ontsluiten van de planning.
+* Er is een samenwerkingsverband tussen de regisseur en de zorgaanbieders.
+* Wij maken gebruik van directe aanroepen en niet van compositions. Hiervoor is gekozen omdat er “live” data
+  nodig is voor de ANW-usecase.
+
+### Woordenlijst
+
+* **Regisseur**: De partij die de koppeling tussen de cliënten en ANW-zorgmedewerker maakt.
+* **Bronsysteem**: Het systeem/zorgaanbieder waar de cliënt in bekend is.
+* **Ontvanger**: Het systeem/zorgaanbieder waar de ANW zorgmedewerker werkt.
+* **ANW-Bronhouder**: is het bronsysteem
+* **ANW-Zorgverlener**: is de ontvanger
+* **ANW-regisseur**: is de regisseur
+* **VC**: Verifiable credential
+* **VP**: Verifiable presentation
+
+### NUTS-adresboek
+
+De regisseur heeft toegang tot het NUTS-adresboek, hierin kunnen de leveranciers voor elke organisatie die onderdeel uit
+zal maken het ANW-netwerk aanduiden dat de betreffende organisatie gebruik maakt van de usecase “ANW-Zorgverlener”
+en “ANW-Bronhouder”. De regisseur is zelf ook te vinden in het adresboek voor de bronhouder en zorgverlener om toegang
+te geven tot deze regisseur met als service “ANW-Regisseur”.
+
+### Regisseur vraagt de cliënt / medewerker gegevens op in alle ECD’s
+
+#### Omschrijving
+
+De regisseur heeft een functionaliteit tot zijn beschikking waarmee hij eenvoudig cliënten en medewerkers met elkaar kan
+koppelen. Dit kan zowel zijn eigen ECD zijn als een applicatie van derden, zie onderstaande afbeelding.
+
+![][image1]  
+Het uitgangspunt is dat de regisseur een eigen NUTS-node heeft draaien. Deze node wordt “getrust” door de NUTS-nodes van
+de verschillende leveranciers. De regisseur kan met behulp van zijn eigen node een JWT-token opvragen om hiermee de
+cliënt- en medewerkergegevens van de verschillende zorgaanbieders op te vragen.
+
+Deze zorgaanbieders vinden de regisseur door binnen het NUTS-adresboek op de usecase “ANW-Bronhouder” en
+“ANW-Zorgverlener” te filteren. Hiervan krijgt hij de endpoints terug die gebruikt kunnen worden om de informatie mee op
+te vragen.
+
+Binnen dit JWT-token wordt gespecificeerd wie de bevraging uitvoert. Wanneer er een samenwerkingsverband is, is het niet
+nodig vertrouwensniveau substantieel te gebruiken. Het bevragen van de cliënt en medewerker gegevens wordt door de
+regisseur geïnitialiseerd. Dit kan op eender welk moment plaatsvinden met een door de regisseur te bepalen frequentie.
+De frequentie is hierbij een redelijk aantal requests per dag.
+
+Op basis van het ontvangen JWT-token kan de zorgaanbieder bepalen tot welke endpoints de regisseur toegang heeft. Dit
+zal voor deze usecase volgende twee: [Endpoints voor data regisseur](#endpoints-voor-data-regisseur)
+
+Als query wordt er een op voorhand afgesproken term “ANW-zorg” meegegeven. Hiermee kan de zorgaanbieder bepalen welke
+cliënten/medewerkers teruggestuurd moeten worden.
+
+Elke leverancier zal zelf in het ECD voorzien dat medewerkers en cliënten als "ANW" cliënt gekenmerkt worden. Afspraak
```

Comment:
not `voorzien` but more of enable since the vendor will not be marking the  employees/clients.